src/components/Amalgamation/BusinessTable.vue

```diff
@@ -15,27 +15,29 @@
       <tbody>
         <tr
           v-for="(item, index) in businesses"
-          :key="item.businessId"
+          :key="item.identifier"
```

Comment:
I initially had this as `:key="index"` but if you delete an entry in the table, most of the indexes stay the same (eg, you go from 0-5 to 0-4) so the table doesn't render correctly. Now it does because each item is indexed by its unique value.

Reply:
I still have work to do in here. Not your concern :)

Reply:
Faced a similar issue when I was running the debug code for the array 🙏